README.md

```diff
@@ -1512,6 +1513,44 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+### SchemaGraph 
+The `SchemaGraph` is an utility class that encodes a database schema and allows for better schema 
+introspection. The  `SchemaGraph` has three main benefits over the GraphQL schema. First, 
```

Comment:
Instead of first/second/last I would make this a succinct && enumerated list (easier to read when skimming)